src/publishPackages/publish.proj

```diff
@@ -21,8 +23,8 @@
     <BuildDependsOn Condition="'$(AzureAccessToken)' != ''">
       $(BuildDependsOn)
       PreparePublish;
```

Comment:
Removed the dependence on older tasks which didn't transport the Manifest Name as expected resulting in outdated Manifest format